include/drivers/dma.h

```diff
@@ -32,7 +32,20 @@ enum dma_channel_direction {
 	MEMORY_TO_MEMORY = 0x0,
 	MEMORY_TO_PERIPHERAL,
 	PERIPHERAL_TO_MEMORY,
-	PERIPHERAL_TO_PERIPHERAL /*only supported in NXP EDMA*/
+	PERIPHERAL_TO_PERIPHERAL,
+	HOST_TO_MEMORY,
```

Comment:
any differences from PERIPHERAL_TI_MEMORY?

Reply:
Hi Hake,

Yes definitely. In some instances there are additional hardware register read/writes needed when the channel is setup this way. This can be shown in the SOF project's drivers, which I intend on porting to Zephyr.

Specifically the HDA DMA driver needs this:
https://github.com/thesofproject/sof/blob/main/src/drivers/intel/hda/hda-dma.c#L385

Reply:
thanks @teburd undersatnd the defferences, just want to clear about whether HOST is a good naming for dma source

Reply:
Ah that's a fair point, I've seen it labeled HMEM and HOSTDDR, really meaning host memory. I'd be ok with HOST_MEM if that's clearer

Reply:
Could you add more information what HOST_TO_MEMORY is supposed to mean in the commit message?

Reply:
thanks. I  am OK with the name as is.